scripts/setup-e2e.sh

```diff
@@ -34,7 +34,8 @@ install_auth_js () {
 
 # Install required node version
 export NVM_DIR="/root/.nvm"
-setup_service node v16.13.2
+setup_service node v16.14.0
+setup_service python 3.6
```

Comment:
just curious, why python will be needed?

Reply:
Some node-gyp scripts during angular installations were failing in CI, the error message didn't like the python version installed